fmt_config/README.md

```diff
@@ -0,0 +1,35 @@
+## YAML Properties for Inline Code Documentation
+
+### `comment1`, `comment2`, `comment3`
+
+- `comment1`: The opening comment delimiter for the language (e.g., `/**` for C++ or Swift). 
+- `comment2`: The comment prefix that begins each line of documentation. For example, in Dart, the comment prefix is `///`, while in Swift it would be `*` if `comment1` is `/**`.
+- `comment3`: The closing comment delimiter for the language (e.g., `*/` for C++ or Swift).
+
+### `summary1` and `summary2`
+
+The summary of the method or function being documented. `summary1` is the opening prefix for the summary, while `summary2` is the closing suffix. These properties are used to enclose the summary text. 
+
+### `tag1` and `tag2`
+
+The prefix and suffix for each documentation tag (e.g., `@param` in Java or `:param:` in Python). 
+
+### `param1`, `param2`, and `param3`
+
+The prefix, suffix, and separator for each parameter in the method or function. For example, in Dart, the prefix is `* [`, the suffix is `] `, and there is no separator. 
+
+### `link1` and `link2`
+
+The prefix and suffix for a link to another part of the documentation or external reference. 
+
+### `ignore`
+
+Any string or keyword that indicates that a particular piece of documentation should be ignored by the parser or generator. This can be useful for excluding internal or experimental functions from the documentation.
+
+### `return1`, `return2`, and `return3`
+
+- `return1`: The prefix that is used on the line immediately preceding the return value of the method or function being documented.
+- `return2`: The separator that is used on the same line as the content.
+- `return3`: The suffix that is used on the line immediately following the return value of the method or function being documented.
```

Comment:
Is there any example of why need to add `return3` option?

Reply:
previously it was 1 and 2, so for c# that would be:
return1: "<returns>"
return2: "</returns>"

But objective-c needs the returns section section to be on one line, for example:
`- Returns: A new RTC Engine object`

Couldn't do that with the existing section, so rather than needing all those "<returns>" to change to "<returns>\n", I added one in the middle, moving return2 to return3. With this input:

```
return1: "RETURN_1"
return2: "RETURN_2"
return3: "RETURN_3"
```

you would get:

/// RETURN_1
/// RETURN_2 <content>
/// RETURN_3